webpack.dev.config.js

```diff
@@ -4,6 +4,6 @@ module.exports = {
     mode: 'development',
     devtool: 'inline-source-map',
     devServer: {
-        contentBase: path.resolve(__dirname, '../dist'),
+        static: path.resolve(__dirname, '../dist'),
```

Comment:
Also it will be nice to fix problem with `dist` created one level above.

Reply:
Oh, right, good catch. Thank you